app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    key_word, read_file, write_file, *_ = command.split()
+    """# After use "*_" instruction no need "len == 3" verif,
```

Comment:
Imagine that the `command `is "cp test". You will encounter an error in this case.

Reply:
I agree, but in this case it is more appropriate to catch ValueError. I don't like the check for len == 3 precisely from the point of view of style, because this way you have to duplicate the operation "command.split()"

Reply:
You don't need to duplicate the command.split() operation. You can do it like this:
....
parts = command.split()
if len(parts) == 3:
    command, read_file, write_file = parts
....